package.json

```
@@ -18,10 +18,12 @@
   "files": [
     "index.js"
   ],
+  "types": "./index.d.ts",
```

the typing is in the `types` folder

```suggestion
  "types": "types/index.d.ts",
```